eng/common/pipelines/replace-relative-links.yml

```diff
@@ -0,0 +1,217 @@
+parameters: 
+  TargetFolder: ''
+  RootFolder: ''
+  BuildSHA: ''
+  RepoId: ''
+
+steps:
+  - task: PythonScript@0
```

Comment:
With our new approach (i.e. not doing a repo ref but instead having it in the repo under eng/common) we should split this out into a python script file so that we can more easily run it locally and debug it. Put the script file under eng/common/scripts.

Reply:
Not as part of this change but we might also consider converting this to powershell at some point. cc @scbedd